groups/prometheus/module-ec2-instances/variables.tf

```diff
@@ -0,0 +1,48 @@
+# Instance
+variable "instance_type" {
+  type = string
+}
+variable "instance_count" {
+  type = number
+}
+variable "ami_version_pattern" {
+  type = string
```

Comment:
Add a `description` argument to each variable declaration throughout this file and other variable files (see [this](https://github.com/companieshouse/smartpan-terraform/blob/master/groups/smartpan/module-smartpan/variables.tf#L4) for an example). Some variables will be self descriptive and not require much in the way of explanation, others can clarify things such as the units a required variable should represent (e.g. an ARN, minutes, gibibytes and so on).